### PR TITLE
[Feature] Partners

### PR DESCRIPTION
**Pull request by KobbyMmo**
## Description

This PR adds Enabling Partner and Newsletter to Homepage

Fixes # ([178713473](https://www.pivotaltracker.com/story/show/178713473))

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image
image
image

### CMS
image

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏾 @KobbyMmo 

-------

QQ feedback, it looks good but it's different from designs:

1. Hover effect missing,
2. Design uses `310px` width for the hover area but the images themselves are not supposed to fill the whole area (using desktop as an example here), and
3. Where is this **Enabling Partners** come from? I only see **Our Partners**.

**Comment by KobbyMmo** [comment]
> 2. Design uses `310px` width for the hover area but the images themselves are not supposed to fill the whole area (using desktop as an example here), and

I might have to give the actual images some padding in photoshpt to get this effect, because all the images are placed differently
 
> 3. Where is this **Enabling Partners** come from? I only see **Our Partners**.

Hah, the designs might have been changed @kilemensi 
image

**Review by kilemensi — COMMENTED** [review]
QQ feedback

1. Something is wrong with the logos,
2. Hovering should change logo colours.
Peek 2021-08-04 09-36

File: src/components/Partners/EnablingPartners.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    </div>
+  );
+}
+
+EnablingPartners.propTypes = {
+  title: PropTypes.string,
+  partners: PropTypes.arrayOf(
+    PropTypes.shape({
+      link: PropTypes.string,
+      logo: PropTypes.string,
```
Comment (KhadijaMahanga) [inline src/components/Partners/EnablingPartners.js:67]: `partners[0].logo` of type `object` supplied to `EnablingPartners`, expected `string`.

File: src/components/Partners/EnablingPartners.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    margin: "0 auto ",
+  },
+}));
+function EnablingPartners({ title, partners, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <Grid container xs={12}>
+        <Grid xs={12}>
```
Comment (KhadijaMahanga) [inline src/components/Partners/EnablingPartners.js:38]:
```suggestion
        <Grid item xs={12}>
```
Failed prop type: The prop `xs` of `Grid` must be used on `item`.

File: src/components/Partners/EnablingPartners.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          <Typography variant="h4" className={classes.title}>
+            {title}
+          </Typography>
+        </Grid>
+        {partners?.map(({ link, logo, name }) => (
+          <Grid item xs={12} md={6}>
```
Comment (KhadijaMahanga) [inline src/components/Partners/EnablingPartners.js:44]: `Warning: Each child in a list should have a unique "key" prop.`

File: src/components/Partners/MainPartner.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  description: {
+    marginTop: typography.pxToRem(40.01),
+  },
+}));
+
+function MainPartner({ link, logo, name, description, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root} item xs={12} md={6}>
```
Comment (KhadijaMahanga) [inline src/components/Partners/MainPartner.js:32]: Warning: Received `true` for a non-boolean attribute `item`.

Was this meant to be a `Grid`?

**Review by KhadijaMahanga — APPROVED** [review]
@KobbyMmo Looks good 🚀 

Couple of props errors & warning :(

The order of logos is not correct, but I guess this is CMS right? Also can we have a screenshot for the CMS in this PR too

**Review by nerdlet — COMMENTED** [review]
@KobbyMmo Loog good! 🚀 
One comment from me is the stay up to date card padding-top and bottom is a bit uneven 
Current implementation 👇🏽 
Screenshot 2021-08-05 at 14 26 55


Designs 👇🏽 
Screenshot 2021-08-05 at 14 27 34

**Comment by kilemensi** [comment]
Please update the components to match [_Developer_ XD](https://xd.adobe.com/view/5aa579ae-4704-4079-abe2-e429d3e5e54f-0ff6/) and not designer XD which talks about Enabling Partners, etc. @KobbyMmo

**Pull request merged**